framework/src/problems/FEProblemBase.C

```diff
@@ -8110,6 +8112,8 @@ FEProblemBase::meshChanged(const bool intermediate_change,
   // "active" and "local" may have been *changed* by refinement and
   // repartitioning done in EquationSystems::reinit().
   _mesh.meshChanged();
+  if (_displaced_problem)
+    _displaced_mesh->meshChanged();
```

Comment:
I don't think this one should be necessary. `es().reinit()` should not touch the displaced mesh